python_src/Dockerfile

```diff
@@ -13,11 +13,18 @@ RUN apt-get install -y libpq-dev gcc curl
 RUN pip install -U pip
 RUN pip install "poetry==1.1.14"
 
-# Copy src dir and pyproject file and create a virtualenv with all needed dependenciies using poetry
-COPY . /lamp/
+# Create a directory for project to live in
+RUN mkdir lamp/
+
+# copy poetry and pyproject files and install dependencies
 WORKDIR /lamp/
```

Comment:
nitpick: I think `WORKDIR` creates a directory, if one doesn't exist, so the `RUN mkdir lamp/` is technically redundant. 

If we want to stick with `RUN mkdir lamp/`, it should probably be an absolute path `/lamp/` instead of relative,  in case our pwd isn't root.